src/i18n/strings/ar.json

```diff
@@ -1056,7 +1056,7 @@
     "Failed to upload profile picture!": "تعذَّر رفع صورة الملف الشخصي!",
     "Show more": "أظهر أكثر",
     "Show less": "أظهر أقل",
-    "Channel: %(channelName)s": "قناة: %(channelName)s",
+    "Channel: <channelLink/>": "قناة: %(channelName)s",
```

Comment:
It seems like you missed replaced both sides here and in various other languages.